package.json

```diff
@@ -467,13 +467,18 @@
         "command": "latex-workshop.closeMathPreviewPanel",
         "title": "Close Math Preview Panel",
         "category": "LaTeX Workshop"
+      },
+      {
+        "command": "latex-workshop.toggleMathPreviewPanel",
+        "title": "Toggle Math Preview Panel",
+        "category": "LaTeX Workshop"
       }
     ],
     "keybindings": [
       {
         "key": "ctrl+alt+m",
         "mac": "cmd+alt+m",
-        "command": "latex-workshop.openMathPreviewPanel"
+        "command": "latex-workshop.toggleMathPreviewPanel"
       },
```

Comment:
We should add the `when` clause for cases `altKeymap` are enabled and disabled.  I have missed it.

Reply:
> I am not sure whether the clause should include the condition of `editorLangId`.

We need it for the `altKeymap` shortcut because it uses `ctrl-l` as prefix, which is also the standard vscode shortcut for expanding line selection. Also, `ctrl-alt-m` is used by the [JSON tools extension](https://marketplace.visualstudio.com/items?itemName=eriklynd.json-tools). So I added the `editorLangId` condition in both shortcuts.